var/spack/repos/builtin/packages/dray/package.py

```diff
@@ -79,6 +80,7 @@ class Dray(Package, CudaPackage):
     depends_on("raja~cuda~openmp~shared", when="~cuda~openmp~shared")
     depends_on("raja~cuda+openmp~shared", when="~cuda+openmp~shared")
 
+    depends_on("umpire@:4.9", when="@:0.1.6")
```

Comment:
@chuckatkins @cyrush I have built `dray@0.1.6` with `umpire@5.1.0`, is there a reason that wouldn't generally work?

Reply:
it's possible umpire 5 is fine, but umpire 6 definitely has API breaking changes (addressed in newer versions of dray)